CONTRIBUTORS.md

```diff
@@ -3,34 +3,35 @@ Chapel Contributors
 
 The following people have contributed to Chapel releases:
 
-Contributors to the Chapel 2.0 release
+Contributors to the Chapel 2.1 release
 --------------------------------------
 * Jade Abraham, [HPE]
-* Paul Cassella, [HPE]
+* [Dan Bonachea], [Lawrence Berkeley National Laboratory]
 * [Brad Chamberlain], [HPE]
 * Soohoon Choi, [HPE]
 * Sarah Coghlan, [HPE]
 * Jeremiah Corrado, [HPE]
+* Mark Dewing, individual contributor
 * Lydia Duncan, [HPE]
 * Daniel Fedorin, [HPE]
 * Michael Ferguson, [HPE]
 * Luca Ferranti, individual contributor
 * Ben Harshbarger, [HPE]
 * John Hartman, [HPE]
 * Guillaume Helbecque, individual contributor
+* Bonnie Hurwitz, [University of Arizona] / [HPE] contractor
 * Engin Kayraklioglu, [HPE]
 * Shreyas Khandekar, [HPE] (former intern from [University of Arizona])
 * Vassily Litvinov, [HPE]
 * David Longnecker, [HPE]
 * Ben McDonald, [HPE] (former intern from [Gonzaga University])
-* Josh Milthorpe, [ORNL]
+* Fábio Malacco Moreira, individual contributor
 * Brandon Neth, [HPE] (former intern from [University of Arizona])
 * Ahmad Rezaii, [HPE]
 * Anna Rift, [HPE]
 * Andy Stone, [HPE] (former [Cray Inc.] intern from [Colorado State University])
 * Michelle Mills Strout, [HPE]
-* Karlon West, [HPE]
-* Tom Westerhout, individual contributor
+* Nicole Vadot, individual contributor
```

Comment:
Thank you for the mention !

Small correction, my affiliation is EPFL SPC : https://spc.epfl.ch/

Reply:
You're welcome Nicole!  

I realize it probably feels inconsistent, but we typically don't mention affiliations in our contributors list unless we have a formal collaboration with the institution in question for fear of getting into a situation where the institution says "We didn't approve this person to work on / contribute to this" or "We think we own this code because the person contributed it while working for us."  That's why, for instance, Dan has an affiliation listed, but most other external contributors don't for this release.  I hope that makes sense and isn't too disappointing.

However, a very easy way around this would be:  If you have a linked in page or webpage that represents you, I'd be happy to link your name to that page, and if it mentions your affiliation, that's a way to convey the same information.

Reply:
Ah yes makes sense, no worries !

In that case you can use the following link : https://people.epfl.ch/nicole.vadot

Many thanks !

Reply:
Added the link, thanks!